gef-ui/templates/includes/application-preview/task-comments.njk

```diff
@@ -5,17 +5,21 @@
 
     {% set bannerHtml %}
 
-      {% if isUkefReviewAvailable and not coverDatesConfirmed %}
+      {% if isUkefReviewAvailable and isUkefReviewPositive and not coverDatesConfirmed %}
```

Comment:
Since this is getting a bit complex now in the template, should we move these 3 conditions do the server, so the template here could be something like

```js
{% if renderReviewDecisionLink %}
```

Thoughts?